scripts/servers.py

```diff
@@ -158,8 +158,10 @@ def managed_dev_appserver(
     Args:
         app_yaml_path: str. Path to the app.yaml file which defines the
             structure of the server.
-        env: dict(str: str) or None. Defines the environment variables for the
-            new process.
+        env: dict(str: str) or None. Defines additional environment
```

Comment:
Docstring may no longer be accurate since it seems the merging part was reverted. You may want to consider keeping that behavior since it's more correct.